design/source/webctrls.pas

```diff
@@ -884,6 +902,7 @@   TWStringGrid = class(TCustomStringGrid)
     property DefaultRowHeight;
     property FixedCols;
     property FixedRows;
+    property PopupMenu;
     property RowCount;
   end;
 
```

Comment:
Can it be that you forgot to publish the ```PopupMenu``` property for all the real WCL controls in ```widgets/```?